dev-docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,142 @@
+# Get started developing for Ubuntu Pro Client
+
+This tutorial will walk you through getting your development environment set up from scratch for contributing to Ubuntu Pro Client.
+
+We generally support the latest interim release and the latest LTS of Ubuntu as development hosts for Ubuntu Pro Client.
+
+> **Note**
+> This tutorial was last tested on Ubuntu 23.10 Mantic Minotaur (mantic).
+
+# From nothing to unit tests
+Open a terminal.
+
+Install necessary host tooling.
+```bash
+sudo apt install git tox pre-commit intltool libapt-pkg-dev sbuild-launchpad-chroot
```

Comment:
I wonder if we should modify our `Makefile` to do that. But if we decide we should, this can be done later

Reply:
I would say we definitely should do that

Reply:
Sure makes sense, we'll just need this for `git` but the rest can be `make install-dev-deps` or something like that

Reply:
this requires a separate PR to add to the Makefile so I'd like to leave it as is for now.

Breaks: create `make install-dev-deps` target to easily install all dev tooling

Reply:
Breaks: create `make install-dev-deps` target to easily install all dev tooling